tests/integration/test_api.py

```diff
@@ -569,3 +569,36 @@ def test_prediction_collection_no_filter(client):
     assert prediction_data[1]["id"] == prediction2.id
     assert prediction_data[1]["type"] == "brand"
     assert prediction_data[1]["value_tag"] == "en:beers"
+
+
+def test_get_unanswered_questions_api(client):
+    result = client.simulate_get("/api/v1/questions/unanswered/")
+
+    assert result.status_code == 200
+    assert result.json == {"questions": [], "status": "no_questions"}
```

Comment:
As for other PR, you should separate this first test from the second.

```suggestion
def test_get_unanswered_questions_api_empty(client):
    result = client.simulate_get("/api/v1/questions/unanswered/")

    assert result.status_code == 200
    assert result.json == {"questions": [], "status": "no_questions"}

def test_get_unanswered_questions_api(client):
```